pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -152,8 +152,6 @@ def test_dt64arr_nat_comparison(self, tz_naive_fixture, box_with_array):
 
 
 class TestDatetime64SeriesComparison:
-    # TODO: moved from tests.series.test_operators; needs cleanup
```

Comment:
this comment pertains to the entire class, not just this test

Reply:
Got it. Moved it back.